styles/prosilver/template/common/contribution_details.html

```diff
@@ -63,6 +63,18 @@
 							<dd>{downloads.INSTALL_LEVEL}</dd>
 						</dl>
 						<!-- ENDIF -->
+						{% if downloads.INSTALL_TO %}
+						<dl>
+							<dt>{{ lang('INSTALL_TO') }}{{ lang('COLON') }}</dt>
```

Comment:
Instead of `}}{{` we should use `~` to concatenate two twig variables together

Reply:
`{{ lang('INSTALL_TO') ~ lang('COLON') }}`

Reply:
Done